js_modules/dagit/packages/core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -364,6 +384,11 @@ const LaunchAssetChoosePartitionsDialogBody: React.FC<Props> = ({
                   selections.length === 2 ? selections[1 - idx].selectedRanges : undefined,
                 ),
               }}
+              isDynamic={assets.some(
```

Comment:
I'm not toooo sure about this logic. @bengotow

Reply:
I think you can simplify this a bit -- up above in this file, we determine a `displayedPartitionDefinition` that is the "primary" one we're going to let you interact with. (Typically there can only be one, but if the selection contains partition definitions that are mapped to each other, it can be valid to have more than one)

I think this can just be `displayedPartitionDefinition?.type === PartitionDefinitionType.DYNAMIC`